tests/TensorFlowSharp.Tests/ArithmeticOperationTests.fs

```diff
@@ -27,3 +27,31 @@ module ArithmeticOperationTests =
         let mulValue = mulTensor.GetValue() :?> float32
 
         Assert.Equal(expected, mulValue)
+
+
+    [<Theory>]
```

Comment:
The test method doesn't have parameters, so should be marked with Fact attribute